examples/music-bot/src/bot.ts

```diff
@@ -10,19 +10,17 @@ import { Track } from './music/track';
 import { MusicSubscription } from './music/subscription';
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
-const { token } = require('../auth.json');
+const { token, deployment_guild_id } = require('../config.json');
 
-const client = new Discord.Client({ intents: ['GUILD_VOICE_STATES', 'GUILD_MESSAGES', 'GUILDS'] });
+const client = new Client({ intents: ['GUILD_VOICE_STATES', 'GUILD_MESSAGES', 'GUILDS'] });
 
-client.on('ready', () => console.log('Ready!'));
+client.on('ready', async () => {
+	console.log('Ready!');
 
-// This contains the setup code for creating slash commands in a guild. The owner of the bot can send "!deploy" to create them.
-client.on('messageCreate', async (message) => {
-	if (!message.guild) return;
-	if (!client.application?.owner) await client.application?.fetch();
-
-	if (message.content.toLowerCase() === '!deploy' && message.author.id === client.application?.owner?.id) {
-		await message.guild.commands.set([
+	const app = await client.application?.fetch();
+	const devGuild = await client.guilds.fetch(deployment_guild_id).catch(() => null);
+	if (app && devGuild) {
+		await devGuild.commands.set([
```

Comment:
It would be great to have some check if the commands are already present in the guild before setting commands again and again on each start of bot.